notebooks/sagemaker/Deploy your own finetuned command-r-0824.ipynb

```diff
@@ -0,0 +1,423 @@
+    "        1. **aws-marketplace:ViewSubscriptions**\n",
+    "        1. **aws-marketplace:Unsubscribe**\n",
+    "        1. **aws-marketplace:Subscribe**  \n",
+    "    2. or your AWS account has a subscription to the packages for <span style=\"background-color: yellow\">Cohere Command R Bring Your Own Finetuning</span>. If so, skip step: [Subscribe to the bring your own finetuning algorithm](#1.-Subscribe-to-the-bring-your-own-finetuning-algorithm)\n",
+    "\n",
+    "## Contents:\n",
+    "\n",
+    "1. [Subscribe to the bring your own finetuning algorithm](#1.-Subscribe-to-the-bring-your-own-finetuning-algorithm)\n",
+    "2. [Preliminary setup](#2.-Preliminary-setup)\n",
+    "3. [Get the merged weights](#3.-Get-the-merged-weights)\n",
+    "4. [Upload the merged weights to S3](#4.-Upload-the-merged-weights-to-S3)\n",
+    "5. [Export the merged weights to the TensorRT-LLM inference engine](#5.-Export-the-merged-weights-to-the-TensorRT-LLM-inference-engine)\n",
+    "6. [Create an endpoint for inference from the exported engine](#6.-Create-an-endpoint-for-inference-from-the-exported-engine)\n",
+    "7. [Perform real-time inference by calling the endpoint](#7.-Perform-real-time-inference-by-calling-the-endpoint)\n",
+    "8. [Delete the endpoint (optional)](#8.-Delete-the-endpoint-(optional))\n",
+    "9. [Unsubscribe to the listing (optional)](#9.-Unsubscribe-to-the-listing-(optional))\n",
+    "\n",
+    "## Usage instructions:\n",
+    "\n",
+    "You can run this notebook one cell at a time (By using Shift+Enter for running a cell)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Subscribe to the bring your own finetuning algorithm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To subscribe to the algorithm:\n",
+    "1. Open the algorithm listing page <span style=\"background-color: yellow\">Cohere Command R Bring Your Own Finetuning</span>.\n",
+    "2. On the AWS Marketplace listing, click on the **Continue to Subscribe** button.\n",
+    "3. On the **Subscribe to this software** page, review and click on **\"Accept Offer\"** if you and your organization agrees with EULA, pricing, and support terms. On the \"Configure and launch\" page, make sure the ARN displayed in your region match with the ARN you will use below."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Preliminary setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install the Python packages you will use below and import them. For example, you can run the command below to install `cohere-aws` if you haven't done so."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!pip install --upgrade cohere-aws"
```

Comment:
No longer need to install the `cohere-aws` after we migrate it to `cohere-python-sdk`